lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -1154,13 +1154,8 @@ protected function prepare_links( $post ) {
 					continue;
 				}
 
-				if ( 'post_tag' === $tax ) {
```

Comment:
Is there going to be some alignment on "tag" (`rest_base`) vs "post_tag" (`slug`)? I get that "post_tag" is "ugly," but who really cares? The API should be data-driven, and not make special cases for personal preferences. The more the API special-cases "tag", the less consistent it becomes. Please, I beg you, use `$tax->slug` everywhere.